styleguide.config.js

```diff
@@ -36,17 +16,10 @@ module.exports = {
     {
       name: 'General',
       components: () => [
-        'src/components/Table/index.jsx',
+        // 'src/components/Table/index.jsx',
```

Comment:
Should this line be removed if it is not required anymore ?

Reply:
Not really, it's only because Table requires stuff directly from cozy-ui and there is a problem with the cozy-client-js required in cozy-ui, in the styleguide.